src/transformers/generation/utils.py

```diff
@@ -136,27 +136,23 @@ class GenerateDecoderOnlyOutput(ModelOutput):
         sequences (`torch.LongTensor` of shape `(batch_size, sequence_length)`):
             The generated sequences. The second dimension (sequence_length) is either equal to `max_length` or shorter
             if all batches finished early due to the `eos_token_id`.
-        scores (`tuple(torch.FloatTensor)` *optional*, returned when `output_scores=True` is passed or when `config.output_scores=True`):
+        scores (`tuple(torch.FloatTensor)` *optional*, returned when `output_scores=True`):
```

Comment:
In our docs we often mention that there are two ways to parameterize `generate` (`generation_config` or pass arg to `generate`). I don't think we need to be verbose here.

Also, setting through `config` is deprecated 😉